### PR TITLE
Feat/#52

### PR DESCRIPTION
**Pull request by Alvin-Voo**
# Purpose / Goal
Added actions to zoom in or out svg panel for issue #52
 
# Type

* [ ] Bug Fix
* [ ] Refactoring / Technology upgrade
* [x] New Feature
* [ ] Documentation
* [ ] Other : | Please Specify |

**Note** : Please ensure that you've read contribution [guidelines](CONTRIBUTING.md) before raising this PR.

### CODE REVIEW

File: tags/workarea.tag.html [hunk 1]
```diff
@@ -108,6 +113,7 @@
                     currentTool.on('drawcancel', function () {});
 
                     currentTool.on('resizedone', function () {
+                        console.log(currentTool.rbox(myCanvas));
```
Comment (amitguptagwl) [inline tags/workarea.tag.html]: console log needs to be removed

File: tags/workarea.tag.html [hunk 2]
```diff
@@ -90,7 +90,12 @@
 
             $("#canvas-container img").css("opacity", appConfig.imageOpacity || 1);
 
-            myCanvas = new SVG('work-canvas').size(opts.img.size.width, opts.img.size.height)/* .panZoom({zoomMin: 0.5, zoomMax: 20}) */ ;
+            //this is needed so that everytime workarea remounts the scale are correct
+            $("#canvas-container img").css({"width": opts.img.size.scaledWidth, "height": opts.img.size.scaledHeight});
```
Comment (amitguptagwl) [inline tags/workarea.tag.html]: It can be clubbed with line above to set opacity property.

**Comment by amitguptagwl** [comment]
I believe you may require to pull latest changes and rebase.

**Comment by Alvin-Voo** [comment]
@amitguptagwl 
Alright, I have merged from master and removed those files. Please see the commits tab.

**Comment by amitguptagwl** [comment]
Thanks @Alvin-Voo

**Pull request merged**